config/common/maintenance.yaml

```diff
@@ -0,0 +1,5 @@
+template:
+  type: http
+  url: https://{{stack_group_config.admincentral_cf_bucket}}.s3.amazonaws.com/aws-infra/v0.3.2/maintenance.yaml
```

Comment:
latest version is [v0.3.6](https://github.com/Sage-Bionetworks/aws-infra/tags) so probably should go with that.